script/riiif_cache_clean.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+#
+# riiif_cache_clean.sh
+# Cron script for maintaining the loris cache size.
+#
+# CAUTION - This script deletes files. Be careful where you point it!
+#
+
+LOG="../tmp/riiif_cache_clean.log"
```

Comment:
@scherztc Let's put the log in our log directory so we don't lose it every time we deploy.  So `log/riiif_cache_clean.log`